components/form/hooks/useForm.ts

```diff
@@ -21,6 +21,24 @@ function toNamePathStr(name: NamePath) {
   return namePath.join('_');
 }
 
+function getFormItemCtrl(node: HTMLElement): HTMLElement {
+  const prefixCls = 'ant-form';
```

Comment:
This will not work when user customize `prefixCls` with ConfigProvider.

Reply:
ok